src/sentry/models/dashboard_widget.py

```diff
@@ -205,6 +229,9 @@ class DashboardWidget(Model):
     widget_type = BoundedPositiveIntegerField(choices=DashboardWidgetTypes.as_choices(), null=True)
     limit = models.IntegerField(null=True)
     detail: models.Field[dict[str, Any], dict[str, Any]] = JSONField(null=True)
+    discover_widget_split = BoundedPositiveIntegerField(
+        choices=DashboardWidgetTypes.as_choices(), null=True
```

Comment:
This could be narrowed but since we're merging / deleting this eventually I'll leave it as the full enum list.